src/content/reference/manifest.mdx

```diff
@@ -699,10 +700,14 @@ The supported keys for containers defined in this section are:
 1. `sync`
 1. `tolerations`
 1. `workdir`
+1. `replicas`
 
 They work the same as for the main container, except for the `command` and `name` keys.
 For `services`, `command` defaults to the command specified in your deployment, instead of `sh`.
 `labels` and `name` are mutually exclusive (for the main container, `name` is mandatory).
+`replicas` specifies the number of replicas for service dev containers.
+If `replicas` is not specified, it defaults to the original deployment replicas.
+If original deployment replicas are zero, it defaults to one replica.
```

Comment:
@AnshulAngaria could we remove this behavior? if original replicas is zero, we should keep zero in the service dev container

Reply:
Sounds good, raised PR to removed this behaviour https://github.com/okteto/okteto/pull/3600